graylog2-web-interface/src/components/events/ColumnRenderers.tsx

```diff
@@ -0,0 +1,168 @@
+  const eventDefinitionTypes = usePluginEntities('eventDefinitionTypes');
+  const plugin = useMemo(() => {
+    if (type === undefined) {
+      return null;
+    }
+
+    return eventDefinitionTypes.find((edt) => edt.type === type);
+  }, [eventDefinitionTypes, type]);
+
+  return <>{(plugin && plugin.displayName) || type}</>;
+};
+
+const PriorityRenderer = ({ priority }: { priority: number }) => <PriorityName priority={priority} />;
+
+const FieldsRenderer = ({ fields }: { fields: Record<string, string> }) => (
+  isEmpty(fields)
+    ? <em>No additional Fields added to this Event.</em>
+    : <EventFields fields={fields} />
+);
+
+const GroupByFieldsRenderer = ({ groupByFields }: {groupByFields: Record<string, string> }) => (
+  isEmpty(groupByFields)
+    ? <em>No group-by fields on this Event.</em>
+    : <EventFields fields={groupByFields} />
+);
+
+const RemediationStepRenderer = ({ eventDefinitionId }: { eventDefinitionId: string }) => {
+  const { meta: { context: eventsContext } } = useMetaDataContext<EventsAdditionalData>();
+  const eventDefinitionContext = eventsContext?.event_definitions?.[eventDefinitionId];
+
+  return (
+    eventDefinitionContext?.remediation_steps ? (
+      <MarkdownPreview show
+                       withFullView
+                       noBorder
+                       noBackground
+                       value={eventDefinitionContext.remediation_steps} />
+    ) : (
+      <em>No remediation steps</em>
+    )
+  );
+};
+
+const StyledDiv = styled.div`
+  cursor: pointer;
+  &:hover {
+    text-decoration: underline;
+  }
+`;
+
+const MessageRenderer = ({ message, eventId }: { message: string, eventId: string }) => {
+  const { toggleSection } = useExpandedSections();
+
+  const toggleExtraSection = () => toggleSection(eventId, 'restFieldsExpandedSection');
+
+  return <StyledDiv onClick={toggleExtraSection}>{message}</StyledDiv>;
+};
+
+const TimeRangeRenderer = ({ eventData }: { eventData: Event}) => eventData.timerange_start && eventData.timerange_end && (
+<div>
+  <Timestamp dateTime={eventData.timerange_start} />
+      &ensp;&mdash;&ensp;
+  <Timestamp dateTime={eventData.timerange_end} />
+</div>
+);
+
+const customColumnRenderers = (permissions: Immutable.List<string>): ColumnRenderers<Event> => ({
+  attributes: {
+    message: {
+      minWidth: 300,
+      renderCell: (_message: string, event) => <MessageRenderer message={_message} eventId={event.id} />,
+    },
+    key: {
+      renderCell: (_key: string) => <span>{_key || <em>No Key set for this Event.</em>}</span>,
+      staticWidth: 200,
+    },
+    id: {
+      staticWidth: 300,
+    },
+    alert: {
+      renderCell: (_alert: boolean) => <EventTypeLabel isAlert={_alert} />,
+      staticWidth: 100,
+    },
+    event_definition_id: {
+      renderCell: (_eventDefinitionId: string) => <EventDefinitionRenderer permissions={permissions} eventDefinitionId={_eventDefinitionId} />,
+    },
```

Comment:
Imo we can define a relating relative `width` here and make the column a bit smaller and maybe give it a slightly larger min width.

![image](https://github.com/user-attachments/assets/07044296-ef87-44a5-85e0-e9283647419e)